src/scheduler/src/test/java/org/cloudfoundry/autoscaler/scheduler/rest/ScheduleRestControllerCreateScheduleAndNofifyScalingEngineTest.java

```diff
@@ -133,6 +133,8 @@ public void testDeleteSchedule() throws Exception {
         mockMvc.perform(
             delete(TestDataSetupHelper.getSchedulerPath(appId)).accept(MediaType.APPLICATION_JSON));
 
+    java.util.concurrent.TimeUnit.SECONDS.sleep(5); // ⏳ Waiting time to ensure, the db had …
```

Comment:
This is not working. See: <https://github.com/cloudfoundry/app-autoscaler-release/actions/runs/12884553093/job/35921129234?pr=3455#step:6:457>